src/__tests__/GameBoard.test.tsx

```diff
@@ -99,6 +99,15 @@ describe('GameBoard component', () => {
         ]}
       />
     );
-    expect(wrapper.find('.winner')).toHaveLength(3);
+
+    const propsArray = boardArray.map(
+      (el, idx: number) =>
+        wrapper
+          .find(BoardSquare)
+          .at(idx)
+          .props().winner
+    );
+
+    expect(propsArray.every(val => val === true)).toBeTruthy();
```

Comment:
Naming: `propsArray` is specifically an array of `winner` props, and `val` is really a `winner`, no?